src/Symfony/Component/Validator/Constraints/RangeValidator.php

```diff
@@ -157,7 +170,7 @@ private function getLimit($propertyPath, $default, Constraint $constraint)
         try {
             return $this->getPropertyAccessor()->getValue($object, $propertyPath);
         } catch (NoSuchPropertyException $e) {
-            throw new ConstraintDefinitionException(sprintf('Invalid property path "%s" provided to "%s" constraint: %s.', $propertyPath, \get_class($constraint), $e->getMessage()), 0, $e);
+            throw new ConstraintDefinitionException(sprintf('Invalid property path "%s" provided to "%s" constraint: "%s".', $propertyPath, \get_class($constraint), $e->getMessage()), 0, $e);
```

Comment:
not sure about this change

Reply:
Fabbot fails on it: <img width="993" alt="Capture d’écran 2020-03-19 à 21 39 20" src="https://user-images.githubusercontent.com/15314293/77113168-98073000-6a2a-11ea-8d1a-3d5c2caf7742.png">

Reply:
This rule has false positives, this looks one of them.

Reply:
Ok 👌